qiskit_metal/toolbox_metal/bounds_for_path_and_poly_tables.py

```diff
@@ -0,0 +1,234 @@
+                                    bool: If there is a key in design with same chip_name as in layer_stack
+                                    Union[None,set]: Either None if the names don't match,
+                                                    or the set of chip_names that can be used.
+        """
+        box_for_ansys = None
+        self.chip_names_matched = None
+        self.valid_chip_names = None
+
+        path_dataframe = self.design.qgeometry.tables['path']
+        poly_dataframe = self.design.qgeometry.tables['poly']
+
+        if box_plus_buffer:
+            # Based on component selection, determine the bounds for box_plus_buffer.
+            frames = None
+            path_and_poly_with_valid_comps = None
+
+            if case == 2:  # One or more components not in QDesign.
+                self.design.logger.warning("One or more components not found.")
+            elif case == 1:  # Render all components
+                frames = [path_dataframe, poly_dataframe]
+            else:  # Strict subset rendered.
+                mask_path = path_dataframe['component'].isin(qcomp_ids)
+                mask_poly = poly_dataframe['component'].isin(qcomp_ids)
+                subset_path_df = path_dataframe[mask_path]
+                subset_poly_df = poly_dataframe[mask_poly]
+                frames = [subset_path_df, subset_poly_df]
+
+            #Concat the frames and then determine the total bounds of all the geometries.
+            # maybe, change name to package_cavity
+            path_and_poly_with_valid_comps = pd.concat(frames,
+                                                       ignore_index=True)
+            minx, miny, maxx, maxy = list(
+                path_and_poly_with_valid_comps['geometry'].total_bounds)
+            # minx, miny, maxx, maxy = list(
+            #     pd.concat(frames, ignore_index=True)['geometry'].total_bounds)
+            # # Add the buffer, using options for renderer.
+            # x_buff = parse_entry(self._options["x_buffer_width_mm"])
+            # y_buff = parse_entry(self._options["y_buffer_width_mm"])
+
+            minx -= x_buff
+            miny -= y_buff
+            maxx += x_buff
+            maxy += y_buff
+            box_for_ansys = (minx, miny, maxx, maxy)
+            return box_for_ansys, path_and_poly_with_valid_comps, self.chip_names_matched, self.valid_chip_names
+        else:  # Incorporate all the chip sizes.
+
+            minx, miny, maxx, maxy = self.get_box_for_ansys()
+            box_for_ansys = (minx, miny, maxx, maxy)
+
+            frames = [path_dataframe, poly_dataframe]
+            path_and_poly_with_valid_comps = pd.concat(frames,
+                                                       ignore_index=True)
+            return box_for_ansys, path_and_poly_with_valid_comps, self.chip_names_matched, self.valid_chip_names
+
+    def get_box_for_ansys(
```

Comment:
Same here also. `get_box_for_ansys` --> `get_stack_xy_bounds`?

Reply:
name changed and pushed.